recipe/meta.yaml

```
@@ -8,31 +8,34 @@ package:
 
 source:
   url: https://github.com/charman2/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 89457f747693a0fdd6c285401af274a2356aab69b3efaa54558d6d92a1cf4ee1
+  sha256: 4147f2da2e7c662970757ba7350dc98a1aeaf72c39c25a531b211dba2e126240
 
 build:
   number: 1
```

This needs to be reset with updating the version

```suggestion
  number: 0
```